docker/entrypoint-initializer.sh

```diff
@@ -24,7 +24,7 @@ echo "Creating Announcement Banner"
 cat <<EOD | python3 manage.py shell
 from dojo.models import Announcement, UserAnnouncement, Dojo_User
 announcement, created = Announcement.objects.get_or_create(id=1)
-announcement.message = '<a href="https://www.defectdojo.com/pricing" target="_blank">Cloud and On-Premise Subscriptions Now Available! Click here for more details</a>'
```

Comment:
Actually, I've been changing this to /contact for the public demo manually

Can we change this to https://defectdojo.com/contact